pipeline/main.py

```diff
@@ -1,279 +0,0 @@
-
-        # compute loss
-        train_loss = loss_fn(logits[train_idx], labels[train_idx])
-        train_acc = torch.sum(logits[train_idx].argmax(dim=1) == labels[train_idx]).item() / len(train_idx)
-
-        valid_loss = loss_fn(logits[val_idx], labels[val_idx])
-        valid_acc = torch.sum(logits[val_idx].argmax(dim=1) == labels[val_idx]).item() / len(val_idx)
-
-        # backward
-        opt.zero_grad()
-        train_loss.backward()
-        opt.step()
-
-        # Print out performance
-        epochs.set_description('Train Acc {:.4f} | Train Loss {:.4f} | Val Acc {:.4f} | Val loss {:.4f}'.format(
-            train_acc, train_loss.item(), valid_acc, valid_loss.item()))
-        
-        if valid_acc < acc:
-            no_improvement += 1
-            if no_improvement == args.early_stopping:
-                print('Early stop.')
-                break
-        else:
-            no_improvement = 0
-            acc = valid_acc
-        
-        scheduler.step()
-
-    # Test with mini batch after all epoch
-    mixhopgcn_model.eval()
-
-    # forward
-    logits = mixhopgcn_model.forward(graph, feats)
-
-    # compute loss
-    test_loss = loss_fn(logits[test_idx], labels[test_idx])
-    test_acc = torch.sum(logits[test_idx].argmax(dim=1) == labels[test_idx]).item() / len(test_idx)
-
-    print("Test Acc {:.4f} | Test loss {:.4f}".format(test_acc, test_loss.item()))
-
-if __name__ == "__main__":
-    """
-    MixHop Model Parameters
-    """
-    parser = argparse.ArgumentParser(description='MixHop GCN')
-
-    # data source params
-    parser.add_argument('--dataset', type=str, default='Cora', help='Name of dataset.')
-    parser.add_argument('--raw-dir', type=str, default='./data', help='Path of dataset.')
-    # cuda params
-    parser.add_argument('--gpu', type=int, default=-1, help='GPU index. Default: -1, using CPU.')
-    # training params
-    parser.add_argument('--epochs', type=int, default=2000, help='Traning epochs.')
-    parser.add_argument('--early-stopping', type=int, default=200, help='Early stopping.')
-    parser.add_argument('--lr', type=float, default=0.5, help='Learning rate.')
-    parser.add_argument('--lamb', type=float, default=5e-4, help='L2 reg.')
-    parser.add_argument('--step-size', type=int, default=40, help='Period of learning rate decay.')
-    parser.add_argument('--gamma', type=float, default=0.01, help='Multiplicative factor of learning rate decay.')
-    # model params
-    parser.add_argument("--hid-dim", type=int, default=60, help='Hidden layer dimensionalities.')
-    parser.add_argument("--num-layers", type=int, default=4, help='Number of GNN layers.')
-    parser.add_argument("--input-dropout", type=float, default=0.7, help='Dropout applied at input layer.')
-    parser.add_argument("--layer-dropout", type=float, default=0.9, help='Dropout applied at hidden layers.')
-    parser.add_argument('--p', nargs='+', type=int, help='Powers list of adjacency matrix.')
```

Comment:
"Powers list of adjacency matrix." -> "List of powers of adjacency matrix."

Reply:
Done.